projects/laji/src/app/+theme/sound-identification/sound-identification-table/sound-identification-table.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit, ChangeDetectionStrategy, ViewChild, TemplateRef, Input } from '@angular/core';
+import { IdentificationData } from '../sound-identification-api';
+import { Observable } from 'rxjs';
+import { LajiApi, LajiApiService } from 'projects/laji/src/app/shared/service/laji-api.service';
+import { TranslateService } from '@ngx-translate/core';
+import { map, shareReplay, tap } from 'rxjs/operators';
+
+@Component({
+  selector: 'laji-sound-identification-table',
+  templateUrl: './sound-identification-table.component.html',
+  styleUrls: ['./sound-identification-table.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SoundIdentificationTableComponent implements OnInit {
+  @ViewChild('clipSpan', { static: true }) public clipSpanTemplate!: TemplateRef<any>;
+  @ViewChild('vernacularName', { static: true }) public vernacularNameTemplate!: TemplateRef<any>;
+
+  @Input() data?: IdentificationData[];
+  @Input() loading = false;
+
+  columns!: any[];
```

Comment:
Could be typed
`import { DatatableColumn } from '../../../shared-modules/datatable/model/datatable-column';`